File: PRAK205-2310817220016-NailaHanifah.py
import math
A = int(input())
B = int(input())
A = float(A)
B = float(B)
C = math.sqrt(B*B - A*A)
alas = C
tinggi = A
keliling = A + B + C
luas = A * C * 1/2
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm^2")

File: PRAK205-2310817220016-NailaHanifah.c
#include <stdio.h>
#include <math.h>
int main() {
    double A, B, C, alas, tinggi, keliling, luas;
    scanf("%lf", &A);
    scanf("%lf", &B);
    C = sqrt(B * B - A * A);
    alas = C;
    tinggi = A;
    keliling = A + B + C;
    luas = A * C * 0.5 ;
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);
    printf("Keliling = %.0fcm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);
    return 0;
}
